examples/bpk-component-modal-v2/examples.tsx

```diff
@@ -227,6 +228,12 @@ const NoFullScreenOnMobileNoTitleExample = () => (
   </ModalContainer>
 );
 
+const NoHeaderExample = () => (
+  <ModalContainer showHeader={false}>
+    This is a default modal. You can put anything you want in here.
```

Comment:
is this the default modal? 😄 the default modal would be when `showHeader` is `true`, no?